pkgs/tools/security/gitjacker/default.nix

```diff
@@ -0,0 +1,40 @@
+{ buildGoModule
+, fetchFromGitHub
+, git
+, lib
+}:
+
+buildGoModule rec {
+  pname = "gitjacker";
+  version = "0.0.2";
+
+  src = fetchFromGitHub {
+    owner = "liamg";
+    repo = "gitjacker";
+    rev = "v${version}";
+    sha256 = "0fg95i2y8sj7dsvqj8mx0k5pps7d0h1i4a3lk85l8jjab4kxx8h9";
+  };
+
+  vendorSha256 = null;
+
+  propagatedBuildInputs = [ git ];
```

Comment:
This should be part of buildGoModule.

Reply:
According to the documentation runtime dependencies of the package should be added to `propagatedBuildInputs`.

Reply:
Oh runtime. Than it is fine I guess.